perfkitbenchmarker/disk.py

```diff
@@ -53,6 +53,8 @@
 REGION = 'region'
 LEGACY_DISK_TYPE = 'legacy_disk_type'
 
+WINDOWS = 'windows'
```

Comment:
This should be referencing the original constant to avoid re-declaration, but the original is in benchmark_spec.py, so that's probably impractical as-is. Ideally, we should drag all the os_type constants out of benchmark_spec.py into a separate module.